src/main/java/htsjdk/samtools/util/SnappyLoader.java

```diff
@@ -57,71 +49,47 @@ public SnappyLoader() {
      * @param verbose if true output a small number of debug messages to System.err
      */
     public SnappyLoader(final boolean verbose) {
-        Constructor<InputStream> inputStreamCtor = null;
-        Constructor<OutputStream> outputStreamCtor = null;
-        Class<Error> snappyErrorClass = null;
+        this(verbose, verbose);
+    }
 
+    /**
+     * Constructs a new SnappyLoader which will check to see if snappy is available in the JVM/library path.
+     * @param logFailures  if true output a message to stderr if snappy loading fails
+     * @param logSuccesses if true output a message to stderr if snappy loading succeeds
+     */
+    public SnappyLoader(final boolean logFailures, final boolean logSuccesses) {
```

Comment:
Do we really need this level of configurability of error messages?  It seems like this should be using the logger instead of printing directly to standard.err anyway.